pkg/traceability/traceability.go

```diff
@@ -35,6 +36,23 @@ const (
 	traceabilityStr               = "traceability"
 )
 
+var traceabilityClients []*Client
+
+// GetClient - returns a random client from the clients array
```

Comment:
Why is there a slice of trace clients, and why would we want a random one?

Reply:
We can have multiple connections to condor, this helps on high volume servers.  random so the metric events are not constantly trying to use the same one.